### DISCUSSION TITLE
Sync your Mastodon bookmarks to LinkAce

### DISCUSSION THREAD

**Discussion by prplecake** [Kovah/LinkAce#634] [General]
Hey all,

I wanted to share my project [mastodon-bookmark-sync](https://github.com/prplecake/mastodon-bookmark-sync) which allows you to sync Mastodon bookmarks to your LinkAce collection.

Support for other bookmarking services is eventually planned, but LinkAce is the currently recommended solution!

**Comment by Kovah** [comment]
Wow that is super cool! Never thought about syncing bookmarks of social media sites to/with LinkAce. Great work!
